components/webpack/webpack.config.js

```diff
@@ -46,7 +49,21 @@ module.exports = async function (env, argv) {
   const resolve = {
     extensions: ['.js', '.tsx', '.ts', '.json'],
     alias: pathMap,
-    modules: [ 'node_modules' ]
+    modules: ['node_modules'],
+    fallback: { 
```

Comment:
Unfortunately, we depend on all these NodeJS polyfills. We were implicitly depending on them before, but now webpack doesn't add them by default.

Reply:
Are they only added to a bundle if it actually uses them? I'm assuming yes but probably best we confirm.